src/TCommandLine.cpp

```diff
@@ -1288,3 +1293,94 @@ void TCommandLine::clearSuggestions()
 {
     commandLineSuggestions.clear();
 }
+
+void TCommandLine::slot_adjustAccessibleNames()
+{
+    bool isMultipleProfilesActive = (mudlet::self()->getHostManager().getHostCount() > 1);
+    const QString hostName{mpHost ? mpHost->getName() : QString()};
+    switch (mType) {
+    case MainCommandLine:
+        if (isMultipleProfilesActive) {
+            setAccessibleName(tr("Input line for \"%1\" profile.",
+                                 // Intentional comment to separate arguments
+                                 "Item name for acccessibilty system. Phrased so that a changable possive suffix "
```

Comment:
Could this grammar be improved upon? It reads awkwardly right now.

Reply:
Do we need to explain it though?

Reply:
Done, *I* think, in 185a4d4 .